### DISCUSSION TITLE
Atlas pH probe degredation?

### DISCUSSION THREAD

**Discussion by johnny2678** [tagyoureit/nodejs-poolController#493] [Q&A]
My acid pump started dumping acid into my pool this week, seemingly to no effect.  Was thinking it was a problem with the injection check valve but that was working fine.  Then did a manual test and reading was way lower (<7) than the atlas probe (8+).  Time for a Fathers Day recalibration?!!

I guess so but probe is new-ish.  Shipped in Sept and installed in Oct of last year - 2021.

Super slow to calibrate.  Here's a 5 minute graph of actual values with a 1m moving average.  When the green line starts bouncing above and below the yellow, then I've got it pretty dialed in. 

![image](https://user-images.githubusercontent.com/8219743/174491080-a6dcbe8e-f774-440f-a99c-6c048cf9e9f9.png)

Atlas's website says response time should be 95% accurate after 1s yet you can see in this graph after 5+ plus mins it's still creeping down.

I had calibrated to 3 points before, less than a year ago.  Cleared calibration and tried again today.

values finally settled after 10-15 mins
7.2 ref solution came in at 7.85
4 ref solution came in at 6.54
10 ref solution came in at 8.71 ???

Here's the pH 10 solution calibration graph.  Took 10 minutes to go from 6 to ~8.5 and still hasn't settled yet. Finally settled around 25 mins.
![image](https://user-images.githubusercontent.com/8219743/174492136-67877ccc-c480-4859-bb7b-94a0e730dc7f.png)

Final REM calibration curve???
![image](https://user-images.githubusercontent.com/8219743/174493320-810a2a17-b2c8-4362-9152-d69752e431e3.png)

Is it my reference solutions?  They are from Atlas but probably 1Y+ old.  I see that Atlas now sells reference packets which I guess I could try.

Also, don't remember if this is normal or not, but there is there supposed to be some type of suspension solution inside the probe? noticed the liquid moving with an air bubble (kind of like a leveling tool).
![image](https://user-images.githubusercontent.com/8219743/174491353-2d6e6143-3573-4e2a-85fa-cbfffbd33759.png)

Anyways, just looking for ideas/feedback.  Cheers! and Happy Fathers Day!

**Comment by rstrouse** [comment]
When you are doing the calibration decrease the read interval to 2000 or less.  This will allow you to watch for stabilization.    Also the fluid in the probe is the pH standard.  If there are bubbles in the glass globe give it a couple of tomahawk chops toward the ground.  If there is air in there it will not read properly.  Follow the calibration instructions in the wiki.  1s reads are only for a calibrated probe.

**Comment by johnny2678** [comment]
just ordered some new calibration solution packs.  Will shorten the read interval, try again, and post back

**Reply by johnny2678** [reply]
just recalibrated... wasn't pretty.

![image](https://user-images.githubusercontent.com/8219743/174665777-fc785897-aaea-4631-b2fc-e72ae9ec4f6c.png)

tried these since I could get them next day... https://www.amazon.com/dp/B06X3TKTBC?psc=1&ref=ppx_yo2ov_dt_b_product_details

thinking my lab probe is faulty now? or worse maybe my controller chip?

**Comment by lockeandev** [comment]
Following this. I installed the industrial pH probe from Atlas in December 2021 and when I opened my pool in April this year, I started noticing a gradually growing discrepancy between the probe reading and the actual pH. Using my pool test kit, I'm now reading 7.2 and the ph Probe is 8.5. Can it really fall out of calibration that fast?

**Reply by johnny2678** [reply]
yea, I installed back in Oct and the difference from the manual test kit kept widening ever since.
Green line is Atlas Probe reading. Yellow line is test kit reading 😬

![image](https://user-images.githubusercontent.com/8219743/174665386-7a36de07-c5b2-47f5-a004-70b5986baf24.png)

Was hoping it was just the calibration but this exercise is proving something else was going on.

My backup plan was just to spring for the industrial probe (running the lab probe now)... oyyyy

**Comment by rstrouse** [comment]
You should calibrate to the standards the numbers should be 7, 4, and 10.  If you are using the standards from Atlas.  Please see the wiki for calibration instructions or the Atlas data sheet.

**Reply by johnny2678** [reply]
well, I did use the standard atlas solutions the first time which produced a pretty awful calibration curve.  Got the idea of trying a different solution from the wiki 

_"Alternate Mid Point Calibration I have found that the standard solutions can degrade over time..."_ 

just thought I would give it a shot.  Might reach out to Atlas...

**Reply by rstrouse** [reply]
Ahhh.  Yes that works if you create your own standards.  I have sone it when my 7 solution wasn’t 7 anymore.  After you calibrate does it parrot the values back at you when you are finished.  For instance, go through the entire calibration then test each solution again.  It should report very close to the value that was used for the calibration for each solution.  Also make sure you rinse between each solution and shake off excess water.  If you get the 7 or 10 buffers in the 4 or the acids from the 4 in either then it will mess them up.

**Reply by johnny2678** [reply]
I have a very meticulous calibration routine.  I have two probe rinsing stations with distilled water and I shake and dry every time.  Then I put it into the reference solution and monitor with grafana.  I graph the actual readings taken every 2s agains a 60s running average and when the actual readings bounce above/below the 60s MA, that's when I hit the calibration #.  Then back to the 2 rinsing stations... repeat.

It's gotta be the probe because it takes anywhere from 10-15 minutes to settle on a number.  Once I calibrate, the probe stays at the correct value - but - to answer your question - after completing the 3 step calibration, the probe does not read the reference value when re-inserting back into the ref solution.  ex. I think when I tested the 4 after calibration it came up in the 5.8-5.9 range.

**Comment by tagyoureit** [comment]
When I had an air bubble in my probe I emailed pictures to Efrem at Atlas and he told me that my "jelly had leaked out".  I don't see the same goobers that are near your probe tip, but Efrem is very quick to answer emails.

![image](https://user-images.githubusercontent.com/1646840/175121508-957f516c-44a9-4e3f-9ef3-a5caaf7433c5.png)

**Reply by johnny2678** [reply]
yep, I'm working with Efram now.  Thinking is right now there might be some RF interference.  I'm going to try and run my probe wires through some SMA bulkhead connectors but I have to wait for the AMZN delivery guy.

**Reply by johnny2678** [reply]
A PSA from Efram:

> "By the way, distilled water is very bad for a pH probe, if you left it in there for a good amount of time, the distilled water will take salt from the probes internal electrolyte solution. This can be reversed by soaking the probe in Kcl solution overnight (storage solution)"

**Reply by rstrouse** [reply]
Thank you for that.  That makes total sense.  I will change the wiki to include that tidbit.  One source of interference can be your salt cell so for shucks you could try to unplug it and see if it makes any difference.

**Reply by johnny2678** [reply]
I tried soaking in storage solution just in case but unfortunately it didn't make a difference.  Lab probe, with no calibration stays pegged at around 12 and doesn't respond to any calibration solutions - stays around 12.  Swapping out for an older consumer pH probe I had lying around reacts like you expect, so I don't think my SWG is causing any trouble.  This is just something that happened to the lab probe with sudden erratic readings.

Efram has been great and we're working through it.  Going to try calibrating at 7 and seeing if that shakes the lab probe loose.

**Reply by orrious** [reply]
The salt cell should not cause issues as long as the carrier board you are using is fully isolated.

**Comment by johnny2678** [comment]
If you would all observe a moment of silence... my lab probe has been officially declared dead by Atlas.  I only got 9 months out of it, which is concerning.

For those that are running the industrial probes, any issues?  How long have then been in service?  Any replacements needed yet?  What type of maintenance are you doing on them?  

Trying to justify the much higher price point.

And then there's this beast, that I'm trying to unsee but not sure I can - https://atlas-scientific.com/ph/industrial-ph-orp-temp-probe-ph/

**Reply by lockeandev** [reply]
well I haven't tried to debug mine as vigorously as you have yet, but my industrial probe has only been in service since the tail end of December 2021, and started giving my issues in april.... so not looking good. 

Does atlas warranty the probes at all?

**Reply by johnny2678** [reply]
> Does atlas warranty the probes at all?

That was my last question to Efram. Their site only states a 30 day warranty for all products but hoping the industrial probes have a longer one.

Also asked about probe maintenance. Trying figure out if there's anything I wasn't doing that I should've been.

**Reply by tagyoureit** [reply]
I bit the bullet and got the ph/orp/temp probe.  I only got it in February but working good so far.  

Given the life expectancy (4 years) vs the lab grade (1.5 years) or consumer grade (1-1.5 years) multiplied by 3 different probes, it seemed to make financial sense.  But the downside is if something goes wrong you have to replace everything.  

I learned the hard way to not let the probe reach freezing temps.  I know have it near my OCP transformer and covered by a garbage bag (again a brilliant idea by @rstrouse).  Direct heat/sun may have a similarly bad effect.

**Reply by johnny2678** [reply]
Pretty sure we didn't have a freeze here in FL in the last week 😂

Not much I can do about the temps here but the flow cell is in a passively vented enclosure. Can't imagine what I could do to get it cooler.

Do you find that the pH values still reads true to a manual Taylor test kit?

Will probably pull the trigger on the combined probe, just want to make sure it will be worth it.

**Reply by johnny2678** [reply]
@lockeandev  have you tried recalibrating?

Curious if it reads more consistently after calibration

**Reply by rstrouse** [reply]
I can confirm that there was no freeze in FL last week as I was in Destin all week.  I have the Industrial probes that are separate.  Mine holds pretty true to the pH from the test kit but it is a judgment call.  That being said it matches perfectly with another pH probe I have.  I do calibrate once a year but do not get any discrepancies like you mention.  It never ends up more than about 1.5/10ths off from the final calibrated value.  This is within the margins and could be simply due to differences with the sample temps.

The water running through my flow cell is typically over 90 in the summer.  I want to make sure I don't feel any warm spots in the water on the off chance somebody brings their kids over to swim.  While I don't get direct sunlight on the pad it is in the weather and we topped out at 108 yesterday.

**Reply by lockeandev** [reply]
@johnny2678 not yet, I hope I will have time this weekend.
